client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -54,6 +70,41 @@ export default connectOptions(
 					<ThanksModal
 						site={ site }
 						source={ 'list' } />
+					{ config.isEnabled( 'manage/themes/upload' ) &&
+						<div>
+							<ThemeUploadCard
+								label={ translate( 'WordPress.com themes' ) }
```

Comment:
Let's capitalize `Themes` in this heading thingy, BTW (and in other instances as well).